From a46ed044b1838b8e4eef510c25d51a9bddc23556 Mon Sep 17 00:00:00 2001
From: Jonathan Nieder <jrnieder@gmail.com>
Date: Fri, 3 Sep 2010 19:51:58 -0500
Subject: Revert "gitweb: Use @diff_opts while using format-patch"

The "commitdiff" feature in gitweb produces diffs for renames that
can't be applied with patch from sid.  Revert the change as a
temporary measure.

After squeeze is released, the change should be reinstated.
Hopefully before then it will be made configurable on a
site-by-site basis, but even if not, GNU patch 2.6.2 or later
should be more widespread by then.

This reverts commit 04794fdc2701f15d79d92805cfe94c9e754e2e05.

Signed-off-by: Jonathan Nieder <jrnieder@gmail.com>
---
 gitweb/gitweb.perl |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/gitweb/gitweb.perl b/gitweb/gitweb.perl
index 84261bb..77a74a3 100755
--- a/gitweb/gitweb.perl
+++ b/gitweb/gitweb.perl
@@ -6343,8 +6343,8 @@ sub git_commitdiff {
 			}
 			push @commit_spec, '--root', $hash;
 		}
-		open $fd, "-|", git_cmd(), "format-patch", @diff_opts,
-			'--encoding=utf8', '--stdout', @commit_spec
+		open $fd, "-|", git_cmd(), "format-patch", '--encoding=utf8',
+			'--stdout', @commit_spec
 			or die_error(500, "Open git-format-patch failed");
 	} else {
 		die_error(400, "Unknown commitdiff format");
-- 
1.7.2.2

